osu.Game.Rulesets.Osu/Mods/OsuModArrange.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using System.Collections.Generic;
+using osu.Framework.Extensions.IEnumerableExtensions;
+using osu.Framework.Graphics;
+using osu.Game.Graphics;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Objects.Drawables;
+using osu.Game.Rulesets.Osu.Objects;
+using OpenTK;
+
+namespace osu.Game.Rulesets.Osu.Mods
+{
+    internal class OsuModArrange : Mod, IApplicableToDrawableHitObjects
+    {
+        public override string Name => "Arrange";
+        public override string ShortenedName => "Arrange";
```

Comment:
Don't know what the shortened name should be, but it should have one. Just leaving it here.

Reply:
I'm not even sure I agree with the name "arrange". It doesn't really describe what is going on that well.

Open to suggestion.

Reply:
Whats about "Flow in"?

Reply:
Discussed a little bit with @peppy today, we're thinking "Transform" (shortname "TR" maybe?), because eventually this is going to have a settings panel to customize the effect further.